smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/RuntimeClientPlugin.java

```diff
@@ -102,6 +93,19 @@ private RuntimeClientPlugin(Builder builder) {
         boolean test(Model model, ServiceShape service, OperationShape operation);
     }
 
+    @FunctionalInterface
+    public interface FunctionParamsSupplier {
+        /**
+         * Returns dynamic parameters for resolve function.
```

Comment:
nit: Can you validate whether this comment block can display properly in Javadoc?

Reply:
Validated!

<details>
<summary>Details</summary>

![Screen Shot 2021-06-24 at 1 59 35 PM](https://user-images.githubusercontent.com/16024985/123331744-85fd2280-d4f4-11eb-8864-aeaaadac967f.png)

</details>